files/d7_make.sh

```diff
@@ -16,52 +18,52 @@ fi
 
 ## Init site if it doesn't exist
 if [[ ! -e $SITEPATH ]]; then
-    sudo d7_init.sh $SITEPATH || exit 1;
+    sudo d7_init.sh "$SITEPATH" || exit 1;
 fi
 
 ## Dump DB before touching anything
-sudo d7_dump.sh $SITEPATH || exit 1;
+sudo d7_dump.sh "$SITEPATH" || exit 1;
 
 ## Delete build dir if it's there
-sudo -u apache rm -rf $SITEPATH/drupal_build
+sudo -u apache rm -rf "$SITEPATH/drupal_build"
 
 ## Build from drush make or die
-sudo -u apache drush -y --working-copy make $MAKEFILE $SITEPATH/drupal_build || exit 1;
+sudo -u apache drush -y --working-copy make "$MAKEFILE" "$SITEPATH/drupal_build" || exit 1;
 
 ## Delete default site in the build
-sudo -u apache rm -rf $SITEPATH/drupal_build/sites/default
+sudo -u apache rm -rf "$SITEPATH/drupal_build/sites/default"
 
 ## Set perms
 echo "Setting permissions of the new build."
-sudo find $SITEPATH/drupal_build -type d -exec chmod u=rwx,g=rx,o= '{}' \;
-sudo find $SITEPATH/drupal_build -type f -exec chmod u=rw,g=r,o= '{}' \;
+sudo find "$SITEPATH/drupal_build" -type d -exec chmod u=rwx,g=rx,o= '{}' \;
+sudo find "$SITEPATH/drupal_build" -type f -exec chmod u=rw,g=r,o= '{}' \;
 
 # Set SELinux or die
 echo "Setting SELinux policy of the new build."
 sudo semanage fcontext -a -t httpd_sys_content_t  "$SITEPATH/drupal_build(/.*)?" || exit 1;
-sudo restorecon -R $SITEPATH/drupal_build || exit 1;
+sudo restorecon -R "$SITEPATH/drupal_build" || exit 1;
 
 ## Set perms
 echo "Setting permissions of default site."
-sudo find $SITEPATH/default -type d -exec chmod u=rwx,g=rx,o= '{}' \;
-sudo find $SITEPATH/default -type f -exec chmod u=rw,g=r,o= '{}' \;
+sudo find "$SITEPATH/default" -type d -exec chmod u=rwx,g=rx,o= '{}' \;
+sudo find "$SITEPATH/default" -type f -exec chmod u=rw,g=r,o= '{}' \;
 
 # Set SELinux or die
 echo "Setting SELinux policy of the default site."
 sudo semanage fcontext -a -t httpd_sys_content_t  "$SITEPATH/default(/.*)?" || exit 1;
 sudo semanage fcontext -a -t httpd_sys_rw_content_t  "$SITEPATH/default/files(/.*)?" || exit 1
-sudo restorecon -R $SITEPATH/default || exit 1;
+sudo restorecon -R "$SITEPATH/default" || exit 1;
 
 ## Link default site folder. Doing this last ensures that our earlier recursive
 ## operations aren't duplicating efforts.
 echo "Linking default site into new build."
-sudo -u apache ln -s $SITEPATH/default $SITEPATH/drupal_build/sites/default
+sudo -u apache ln -s "$SITEPATH/default" "$SITEPATH/drupal_build/sites/default"
 
 ## Now that everything is ready, do the swap
 echo "Placing new build."
-sudo rm -rf $SITEPATH/drupal_bak
```

Comment:
Need to verify that apache user can/should be able to do this in a production environment. I recall specifically excluding apache from some operations for security reasons. Of course, I don't recall what those reasons are.  We should probably make sure to add a comment with the reason for excluding apache from here on out so we don't have to remember.